opentracing-spring-web-itest/common/src/test/java/io/opentracing/contrib/spring/web/interceptor/itest/common/AbstractBaseITests.java

```diff
@@ -418,6 +417,16 @@ public void testControllerView() {
         assertLogEvents(span.logEntries(), Arrays.asList("preHandle", "afterCompletion"));
     }
 
+    @Test
+    public void testExcludePattern() throws InterruptedException {
+        {
+            getRestTemplate().getForEntity("/health", String.class);
+            Thread.sleep(505);
```

Comment:
I know that you will ask :+1: . Just making sure that span is not created.

> And more intriguingly, why 505? 

It's a nice number but maybe 100ms would be enough.